### ISSUE TITLE
Suggestion for Silhouette Studio Munki Recipe

### ISSUE THREAD

**Issue by JCSmillie** [autopkg/joshua-d-miller-recipes#90]
Not necessarily an issue, but maybe a suggestion?

I'm using your Munki Recipe for Silhouette Studio.  Our faculty are Standard users on their Macs so they don't have the ability to authenticate the first time the app runs to make the folders it wants.  I've added this to my post install scripts in Munki to make those folders for them:

`mkdir "/Library/Application Support/com.aspexsoftware.Silhouette_Studio"
mkdir "/Library/Application Support/com.aspexsoftware.Silhouette_Studio.license"
chmod -R 777 "/Library/Application Support/com.aspexsoftware.Silhouette_Studio"
chmod -R 777 "/Library/Application Support/com.aspexsoftware.Silhouette_Studio.license"`

The app opens now.  It asks you about installing some content for Previews of files which I haven't figured out what its doing yet, but this at least gets you to the point, as a non-admin, that you can run the software.

**Comment by joshua-d-miller** [comment]
Hello @JCSmillie,

This is great however I'm a little nervous about granted full-out rights to the files in question. It looks to me like it might just be the license files but I'm curious if you did something like 755 if that would also work?

Thanks!

**Comment by JCSmillie** [comment]
Sadly 777 is how the folders are created by the app itself if you are an administrator or you give admin auth the first time the program runs.  If you change it to 755 and then open the app you get an error about can't read device hash and then the app goes into an infinite open/crash loop.  I tried moving these to the users own home directory out of curiosity if it would look to the users own area for them but no luck.

FWIW I did reach out to support about this.  Why does the app need admin privs?  What is it doing with them?  Why can't you write in the users home dir?  etc, etc, etc.  Got some lip service back and eventually it went silent.
